src/components/CreateChallenge.tsx

```diff
@@ -0,0 +1,49 @@
+import _ from 'lodash';
+import React from 'react';
+
+import Button from './Button';
+
+interface Props {
+  createChallenge: (challenge: any) => void;
+  currentPlayer?: {
```

Comment:
I think we know by this stage in the app that the `currentPlayer` must exist.

Reply:
This is a bigger change, as the parent component doesn't know for sure that `currentPlayer` exists ...